sympy/core/basic.py

```diff
@@ -1716,6 +1716,35 @@ def _exec_constructor_postprocessors(cls, obj):
 
         return obj
 
+    def at_reals(self, x, side):
```

Comment:
Should this be on Expr?

Reply:
That is what I thought first but then the Tuples in Piecewise were not included.